src/core/evaluator.js

```diff
@@ -723,29 +723,70 @@ var PartialEvaluator = (function PartialEvaluatorClosure() {
           return errorFont();
         }
       }
+      let fallbackFont = false;
+      let fallbackFontProperties = false;
       if (!fontRef) {
-        warn('fontRef not available');
-        return errorFont();
+        // warn('fontRef not available');
+        // return errorFont();
```

Comment:
Please note that commented-out code isn't allowed in general, and in this case you very much want to print the warning since there's simply no guarantee that things will actually work particularly well in general with this sort work-around.

Also, an sending an `UnsupportedFeature` notification would seem appropriate as well.